src/status_im/contexts/wallet/home/view.cljs

```diff
@@ -71,9 +72,53 @@
         tokens-loading?                (rf/sub [:wallet/home-tokens-loading?])
         networks                       (rf/sub [:wallet/selected-network-details])
         account-cards-data             (rf/sub [:wallet/account-cards-data])
+        testnet-mode?                  (rf/sub [:profile/test-networks-enabled?])
         cards                          (conj account-cards-data (new-account-card-data))
         [init-loaded? set-init-loaded] (rn/use-state false)
         {:keys [formatted-balance]}    (rf/sub [:wallet/aggregated-token-values-and-balance])
+        multiple-accounts?             (> (count account-cards-data) 1)
```

Comment:
could we isolate these changes into a separate component (`quo/wallet-ctas` with all the props it needs) to avoid this one getting too big?

Reply:
I'd recommend a separate component as well. This screen will only grow in future.